### ISSUE TITLE
Don't generate PROC prologue/epilogue when not necessary

### ISSUE THREAD

**Issue by db4** [Terraspace/UASM#119]
This code
```
.CODE

option WIN64: 1

func PROC uses rbx
     ret
func ENDP

END
```
is compiled as
```
func:
  0000000000000000: 48 55              push        rbp
  0000000000000002: 53                 push        rbx
  0000000000000003: 48 83 EC 08        sub         rsp,8
  0000000000000007: 48 8B EC           mov         rbp,rsp
  000000000000000A: 48 8D 65 08        lea         rsp,[rbp+8]
  000000000000000E: 5B                 pop         rbx
  000000000000000F: 5D                 pop         rbp
  0000000000000010: C3                 ret
```
why prologue/epilogue is generated? jwasm 2.12 used to compile it correctly:
```
func:
  0000000000000000: 53                 push        rbx
  0000000000000001: 5B                 pop         rbx
  0000000000000002: C3                 ret
```

**Comment by john-terraspace** [comment]
I’ll have a look at this one, this is probably due to the fact that win64:1 makes no attempt to optimise the prologue so doesn’t take into consideration that the single uses aligns the stack already.

**Comment by john-terraspace** [comment]
Fixed in 2.50
